test_common/jest.config.js

```diff
@@ -37,27 +15,17 @@ const config = {
       statements: 100,
     },
   },
-  moduleDirectories: ['node_modules', 'client', 'server'],
-  moduleFileExtensions: ['js', 'feature', 'ts'],
-  testEnvironment: 'jsdom',
-  testMatch: ['**/*.spec.ts', '**/*.steps.ts'],
-  testPathIgnorePatterns: ['/e2e/'],
-  transform: {
-    '^.+\\.(ts|tsx)$': 'ts-jest',
-    '^.+\\.(js|jsx|ts|tsx)$': [
-      'babel-jest',
-      { configFile: './test_common/test.babel.js' },
-    ],
-  },
-  moduleNameMapper: {
-    ...jestModuleMapper,
-  },
-  setupFiles: ['<rootDir>/test_common/jest_cucumber_support/index.ts'],
-  globals: {
-    'ts-jest': {
-      tsconfig: '<rootDir>/test_common/tsconfig.test.json',
-    },
-  },
+  collectCoverageFrom: [
+    '(client|server)/**/*.{js,ts,jsx,tsx}',
+    '!**/index.{js,ts,jsx,tsx}',
+    '!**/*.steps.*',
+    '!**/*.d.ts',
+    '!**/test_common/**',
+    '!**/server/*',
+    '!**/Home.tsx',
+  ],
+  projects: [
```

Comment:
Project support to allow jest to run once, but run the two sets of tests for client/server. This is needed because the client and server have different tsconfigs.